examples/jupyter/nuclear-data-resonance-covariance.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import os\n",
+    "from pprint import pprint\n",
+    "import shutil\n",
+    "import subprocess\n",
+    "import urllib.request\n",
+    "\n",
+    "import h5py\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.cm\n",
+    "from matplotlib.patches import Rectangle\n",
+    "\n",
+    "import openmc.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ENDF: Resonance Covariance Data\n",
+    "\n",
+    "We can also load the resonance covariance data contained within File 32 of ENDF. Let's download the ENDF/B-VII.1 evaluation for $^{157}$Gd and load it in:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<IncidentNeutron: Gd157>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Download ENDF file\n",
+    "url = 'https://t2.lanl.gov/nis/data/data/ENDFB-VII.1-neutron/Gd/157'\n",
+    "filename, headers = urllib.request.urlretrieve(url, 'gd157.endf')\n",
+    "\n",
+    "# Load into memory\n",
+    "gd157_endf = openmc.data.IncidentNeutron.from_endf(filename, covariance = True)\n",
+    "gd157_endf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can access the parameters contained within File 32 in a similar manner to the File 2 parameters from before. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    energy    J  neutronWidth  captureWidth  fissionWidthA  fissionWidthB  L\n",
+      "0   0.0314  2.0      0.000474        0.1072            0.0            0.0  0\n",
+      "1   2.8250  2.0      0.000345        0.0970            0.0            0.0  0\n",
+      "2  16.2400  1.0      0.000400        0.0910            0.0            0.0  0\n",
+      "3  16.7700  2.0      0.012800        0.0805            0.0            0.0  0\n",
+      "4  20.5600  2.0      0.011360        0.0880            0.0            0.0  0\n"
+     ]
+    }
+   ],
+   "source": [
+    "first_five = gd157_endf.resonance_covariance.ranges[0].parameters[:5]\n",
+    "print(first_five)"
```

Comment:
Here you don't really need to assign to a variable and then print it. Just running:
```Python
gd157_endf.resonance_covariance.ranges[0].parameters[:5]
```
would be sufficient because Jupyter notebook will show the result of the expression